### ISSUE TITLE
In your code, I don't understand what 'logits' and 'targets' mean, could you explain it? Thanks a lot!

### ISSUE THREAD

**Issue by ChuXX325** [autonomousvision/plant#27]
In your code, I don't understand what 'logits' and 'targets' mean, could you explain it? Thanks a lot!

![Image](https://github.com/user-attachments/assets/71c255b6-74b8-4d6e-bfe6-73b4633bf230)
